README.md

```diff
@@ -23,10 +32,10 @@ _"... while some certainly had a defensive purpose, the use of others is not cle
 https://en.wikipedia.org/wiki/Talaiot
 
 # Table of Contents
-1. [Setup Plugin](#setup-plugin)
+1. [Setup Plugin](#setup)
 2. [Snapshots](#snapshots)
 3. [Basic Configuration](#basic-configuration)
-4. [Talaiot Extension](#talaiot-extension)
+4. [Talaiot](#talaiot-extension)
 5. [Example: Analyzing Data provided by Talaiot](#example)
 6. [Other Plugins](#plugins)
```

Comment:
[Title has changed](https://github.com/cdsap/Talaiot/pull/253/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R567) to "Other Existing Libraries". Should we update it in here as well?

Reply:
done

Reply:
done